widgetssdk/src/main/res/layout/call_buttons_layout.xml

```diff
@@ -134,13 +134,13 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginBottom="@dimen/glia_small"
-        android:contentDescription="@string/glia_call_minimize_content_description"
         android:src="?attr/gliaIconCallMinimize"
         app:backgroundTint="@color/call_fab_bg_color_states"
         app:layout_constraintBottom_toTopOf="@id/minimize_button_label"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@id/speaker_button"
         app:rippleColor="@color/call_fab_ripple_color_states"
-        app:tint="@color/call_fab_icon_color_states" />
+        app:tint="@color/call_fab_icon_color_states"
+        tools:contentDescription="@string/engagement_minimize_video_button" />
```

Comment:
tools enables the resource to be seen while editing the layout file. It does not make sense to have tools:contentDescription defined, because content description can not be tested outside runtime

Reply:
Yes, indeed. Fixed 85f24d2145da80062e22debedebdd4ee8cfb153d

Reply:
Also, I removed it here 190820edadad229cc36c26f5a2f9babba6e051e7